pysc2/bin/replay_actions.py

```diff
@@ -42,97 +42,43 @@
 from s2clientprotocol import common_pb2 as sc_common
 from s2clientprotocol import sc2api_pb2 as sc_pb
 
+import importlib
+import json
+import sys
+
 FLAGS = flags.FLAGS
 flags.DEFINE_integer("parallel", 1, "How many instances to run in parallel.")
 flags.DEFINE_integer("step_mul", 8, "How many game steps per observation.")
 flags.DEFINE_string("replays", None, "Path to a directory of replays.")
+flags.DEFINE_string("parser", "pysc2.replay_parsers.base_parser.BaseParser",
+                    "Which agent to run")
+flags.DEFINE_string("data_dir", "C:/",
+                    "Path to directory to save replay data from replay parser")
+flags.DEFINE_integer("screen_resolution", 16,
+                     "Resolution for screen feature layers.")
+flags.DEFINE_integer("minimap_resolution", 16,
+                     "Resolution for minimap feature layers.")
 flags.mark_flag_as_required("replays")
+FLAGS(sys.argv)
```

Comment:
I needed this to get around Issue #77

Reply:
This should be dealt with by app.run and shouldn't be explicitly needed.